.travis.yml

```diff
@@ -2,10 +2,54 @@ language: ruby
 rvm:
   - 2.6.3
 
-install:
-  - bundle install
-  - npm install eslint --save-dev
-  - ./node_modules/.bin/eslint src/main/content/_assets/js/*.js
+stages:
+  - name: Linting Test
+    if: type IN (push, pull_request, api)
+  - name: Fork Deploy
+    if: (type IN (api)) AND (repo != kabanero-io/kabanero-website)
+  - name: Production Deploy
+    if: (type IN (api)) AND (repo == kabanero-io/kabanero-website)
 
-script:
-  - ./scripts/build_jekyll_maven.sh
\ No newline at end of file
+jobs:
+  include:
+    - stage: "Linting Test"
+      name: "Running lint testing"
+      env: TRAVIS_TEST=true
+      install:
+        - bundle install
+        - npm install eslint --save-dev
+        - ./node_modules/.bin/eslint src/main/content/_assets/js/*.js
+      script:
+        - ./scripts/build_jekyll_maven.sh
+
+    - stage: "Fork Deploy" 
+      name: "Deploy to IBM CF from forked repository"
+      env: JEKYLL_DRAFT_GUIDES=true JEKYLL_DRAFT_BLOGS=true
+      install:
+        - bundle install
+      skip_cleanup: true
+      on: 
+        repo: $TRAVIS_REPO_SLUG
+        branch: $TRAVIS_BRANCH
+      script:
+        - curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
+        - ./scripts/build_clone_blogs.sh $BLOGS_GIT_URL $BLOGS_GIT_REVISION
+        - ./scripts/build_clone_docs.sh  $DOCS_GIT_URL  $DOCS_GIT_REVISION
+        - ./scripts/build.sh
+        - ./scripts/ibmcloud-cf-push.sh ${TRAVIS_REPO_SLUG%/*}-kabanero
+
+    - stage: "Production Deploy"
```

Comment:
we really just need 1 deploy stage that is generalized for deployment to any account